### PR TITLE
Make Ogmios compatible with django's cached loader.

### PR DESCRIPTION
**Pull request by jxcl**

### CODE REVIEW

File: ogmios/__init__.py [hunk 1]
```diff
@@ -65,10 +65,19 @@ def get_template_source(self):
 
         for engine in engine_list:
             for loader in engine.engine.template_loaders:
-                try:
-                    return loader.load_template_source(self.filename)[0]
-                except TemplateDoesNotExist:
-                    pass
+                if hasattr(loader, 'get_template_sources'):
```
Comment (acatton) [inline ogmios/__init__.py]: why is it get_template_sources. This needs a comment

File: ogmios/__init__.py [hunk 2]
```diff
@@ -65,10 +65,22 @@ def get_template_source(self):
 
         for engine in engine_list:
             for loader in engine.engine.template_loaders:
-                try:
-                    return loader.load_template_source(self.filename)[0]
-                except TemplateDoesNotExist:
-                    pass
+                # Check for the cached template loader
+                if hasattr(loader, 'get_template_sources'):
```
Comment (gavinwahl) [inline ogmios/__init__.py:69]: Why do you check for one attribute but then use another?

Comment (jxcl) [inline ogmios/__init__.py:69]: see lines 70 and 71.

Comment (gavinwahl) [inline ogmios/__init__.py:69]: Yes, I read the comment. If you want to special case the cached template loader why don't you check for the cached template loader?

**Pull request merged**